### PR TITLE
Update client and schema

### PR DESCRIPTION
**Pull request by Fenthick**
Issue: KNTK-361

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ install: build
 	mv ${BINARY} ~/.terraform.d/plugins/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/${OS_ARCH}
 
 test:
-	TEST_API_SERVER_ENDPOINT=${TEST_API_SERVER_ENDPOINT} ./tools/run_test_api_server.sh
+	TEST_API_SERVER_ENDPOINT=${TEST_API_SERVER_ENDPOINT} ./tools/start_test_api_server.sh
```
Comment (mmac-m3a) [inline Makefile:19]: If the server really needs warm-up time, let's include it in the `start_test_api_server.sh` script.

**Review by mmac-m3a — APPROVED** [review]
Looks good except for minor details that you can address at your leisure. Marking of attributes as read-only is probably not entirely correct, but we can address it separately.

**Comment by mmac-m3a** [comment]
I'll merge the code, but not cut a new release until I verify the read-only marking of test attributes.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 # This GitHub action can publish assets for release when a tag is created.
-# Currently its setup to run on any tag that matches the pattern "v*" (ie. v0.1.0).
+# Currently, its setup to run on any tag that matches the pattern "v*" (ie. v0.1.0).
```
[the remark this answers is not in the document]
Comment (Fenthick) [reply .github/workflows/release.yml:2]: I'll fix that in next PR

File: docs/data-sources/agent.md [hunk 3]
```diff
@@ -31,10 +31,12 @@ output "agent_get" {
 
 ### Read-Only
 
+- **agent_impl** (String)
 - **alias** (String)
 - **asn** (Number)
 - **challenge** (String)
```
[the remark this answers is not in the document]
Comment (Fenthick) [reply docs/data-sources/agent.md:37]: I'll fix that in next PR

File: Makefile (hunk 1 above)
Comment (Fenthick) [reply Makefile:19]: I'll fix that in next PR

**Pull request merged**